ui-admin/src/forms/designer/PagesList.tsx

```diff
@@ -68,6 +68,22 @@ export const PagesList = (props: PagesListProps) => {
                   }
                 }}
               />
+              <IconButton
+                aria-label="Delete this page"
+                className="ms-2"
+                disabled={readOnly || pages.length === 1}
+                icon={faChevronDown}
```

Comment:
I think you want `faTimes` here

Reply:
Oops. Fixed in https://github.com/broadinstitute/juniper/pull/458/commits/38cc200d71a44f0fd3f7f8ac259e7e44ac67991c.